core/templates/dev/head/profile/signup.html

```diff
@@ -80,62 +81,50 @@ <h2 class="oppia-signup-page-title">Complete Your Registration</h2>
 
         <hr>
 
-        <p style="font-size: 1em;">
-          The {{SITE_NAME}} site is an open commons of learning resources, and all material
-          on it is freely reusable and shareable.
+        <p style="font-size: 1em;" translate translate-value-sitename="{{SITE_NAME}}">
```

Comment:
One thought -- maybe this has been fixed in a future update? angular-translate is now v2.7.2 and some of the release notes look like they might pertain to this.

    https://github.com/angular-translate/angular-translate/releases

May be worth experimenting, just to see if the issue goes away.